.pre-commit-config.yaml

```
@@ -18,18 +18,18 @@ repos:
           - ggmosaic
           - ggplot2
           - shiny
-          - shinyTree
           - insightsengineering/teal
+          - insightsengineering/teal.transform
```

```suggestion
          - teal
          - teal.transform
```
Shouldn't this be updated now that the packages are available on CRAN?